### ISSUE TITLE
Runtime questions

### ISSUE THREAD

**Issue by bremsen** [morrislab/phylowgs#105]
Hello,

Thanks for this great tool! 

1) How does the run-time scale with number of CNVs?
I understand that CNVs are converted to pseudo-SSMs, so do CNVs effectively increase the number of SSMs?

2) Can I use PhyloWGS on CNVs alone? I know we can do SSMs alone with no CNVs, but what about the other way?

Thanks!

**Comment by AmitDeshwar** [comment]
Hi!
1. Yes, each CNV is equivalent to adding an additional SNV.  In addition, each SNV affected by a CNV requires a slower computation than an SNV unaffected by CNVs.
2. I don't think there's any technical restriction on this, but I've never tested this.
3. How many CNVs do you have? We've never used data with thousands of CNVs, so I don't think we've ever implemented a subsampling for CNVs.  You can always manually subsample rows in the resulting processed CNV file

**Comment by bremsen** [comment]
Hi Amit,

Thank you for the responses. 

Is there a way to get a list of SNVs and CNVs in each inferred subclone? The tree viewer tells us how many SNVs and CNVs but not which ones specifically.

Thank you!

**Comment by AmitDeshwar** [comment]
Hi!
You can still use CNVs that don't overlap any SNVs to make a more complete genotype for each cluster, because CNVs are clustered just like SNVs.  If you're only interested in the SNVs they're probably not needed.
Yes, the identity of the mutations assigned to each cluster in each tree are found in the trees.json file.

**Comment by bremsen** [comment]
Hi Amit,

Another question: when using the parser on the Titan CNV segments, it seems that the parser tries to assign each segment a unique CNV ID (c0, c1, etc.)

I see that one of the cnv IDs corresponds to many segments (there are 300+ physical CNVs assigned to c0). Is this intended? I don't see any obvious relationship between these segments. Why do they all have the same CNV ID? 

Thanks!

**Comment by bremsen** [comment]
Hi Amit,

Thank you for your responses. I have yet another question. 

Do you know why the parser groups many CNV segments under a single CNV ID? I see one CNV ID (c0) has over a thousand CNVs under "physical CNVs".

Thank you.

**Comment by AmitDeshwar** [comment]
@jwintersinger  Is this a clonal CNV parsing thing?

**Comment by quaidmorris** [comment]
Yep, we group all the clonal CNAs into one single big CNA to make sure that
they all go into the same node.
